src/openrct2/object/DefaultObjects.cpp

```diff
@@ -165,4 +166,20 @@ namespace OpenRCT2
         "rct2.terrain_edge.wood_black",
         "rct2.terrain_edge.ice",
     };
+
+    constexpr std::array<std::string_view, 12> kDefaultCampaignObjects = {
+        "rct1.campaign.base_game",
+        "rct1aa.campaign.added_attractions",
+        "rct1ll.campaign.loopy_landscapes",
+        "rct2.campaign.base_game",
+        "rct2ww.campaign.wacky_worlds",
+        "rct2tt.campaign.time_twister",
+        "rct2.campaign.real_parks",
+        "rct2.campaign.build_your_own",
+        "openrct2.campaign.competitions",
+        "openrct2.campaign.uces",
+        "openrct2.campaign.original_dlc",
+        "openrct2.campaign.rct_classic",
```

Comment:
```suggestion
        "rct2.campaign.rct_classic",
```

I think it should be moved up as well, right before `rct2.campaign.base_game` would be a good spot so it sits in between RCT1 and RCT2.

Reply:
Done, thanks for catching this.